distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -309,3 +309,50 @@ Here are some helpful suggestions for setting up data backups and recoveries:
 * Run a scheduler program that calls the backup URL on a timed basis.
 * Put indicators in place that can detect when data corruption may have occurred.
 * Testing a backup before recovery is possible. A replicated "staging" Solr server instance can be stood up, and the backup can be copied to that system for testing before moving it to the "production" system.
+
+=== High Availability (Optional)
+
+==== SolrCloud
+
+SolrCloud provides distributed capabilities of a highly available, fault tolerant cluster of Solr servers.
+
+===== Prerequisites
+
+Puppet installed and configured.
+
+JDK 6 or greater.
+
+====== Installing and Uninstalling
+
+. `cd /tmp`
+. `wget http://archive.apache.org/dist/lucene/solr/<VERSION>/solr<VERSION>.tgz`
+. Extract the installation script `tar xvf solr-6.0.0.tgz solr-6.0.0/bin/install_solr_service.sh`
+. Execute the installation script ``/tmp/solr-6.0.0/bin/install_solr_service.sh solr-6.0.0.tgz`
+. Execute ``./solr start -e cloud -noprompt -z localhost:<PORT_NUMBER>``
+. Modify ``/opt/solr/bin/solr.in.sh` for your environment's needs.
+The main things you want to change are *ZK_HOST* and *SOLR_HOST*.
+For a production environment, you will also want to change memory settings, etc.
```

Comment:
Should avoid "you." Maybe just a list of attributes to change and possible values?